openmeter/ent/schema/billing.go

```diff
@@ -680,6 +680,9 @@ func (BillingInvoice) Fields() []ent.Field {
 		field.Enum("status").
 			GoType(billing.InvoiceStatus("")),
 
+		field.JSON("status_details_cache", billing.InvoiceStatusDetails{}).
```

Comment:
is it called cache because it's the last synced status?
If yes, how about calling it: `external_status` or `invoice_provider_status`

Reply:
The UpdatedAt reflects that information. Whenever we need to touch the invoice, we need to recalculate the status details either ways.